config/elasticloadbalancing/config.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package elasticloadbalancing
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+)
+
+// Configure adds configurations for elasticloadbalancing group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_lb", func(r *config.Resource) {
+		r.Kind = "LoadBalancer"
+		r.ShortGroup = "elasticloadbalancing"
```

Comment:
should we move to default resources given there are multiple which could be handled commonly?